ReactQt/runtime/src/reactimagemanager.cpp

```diff
@@ -11,348 +11,276 @@
-    case PreserveAspectCrop: return "cover";
-    default: return "";
-  }
-  return "";
+    int resizeMode = m_object->property("fillMode").toInt();
+    switch (resizeMode)
+    {
+    case Stretch:
+        return "stretch";
+    case PreserveAspectFit:
+        return "contain";
+    case PreserveAspectCrop:
+        return "cover";
+    default:
+        return "";
+    }
+    return "";
 }
 
 void ImagePropertyHandler::setResizeMode(const QString& resizeMode)
 {
-  if (resizeMode == "stretch") {
-    m_object->setProperty("resizeMode", QVariant::fromValue(int(Stretch)));
-  } else if (resizeMode == "contain") {
-    m_object->setProperty("resizeMode", QVariant::fromValue(int(PreserveAspectFit)));
-  } else if (resizeMode == "cover") {
-    m_object->setProperty("resizeMode", QVariant::fromValue(int(PreserveAspectCrop)));
-  }
+    if (resizeMode == "stretch")
+    {
+        m_object->setProperty("resizeMode", QVariant::fromValue(int(Stretch)));
+    }
+    else if (resizeMode == "contain")
+    {
+        m_object->setProperty("resizeMode", QVariant::fromValue(int(PreserveAspectFit)));
+    }
+    else if (resizeMode == "cover")
+    {
+        m_object->setProperty("resizeMode", QVariant::fromValue(int(PreserveAspectCrop)));
+    }
 }
 
-QString ImagePropertyHandler::backfaceVisibility() const
-{
-  return QString();
-}
+QString ImagePropertyHandler::backfaceVisibility() const { return QString(); }
 
-void ImagePropertyHandler::setBackfaceVisibility(const QString& backfaceVisibility)
-{
-}
+void ImagePropertyHandler::setBackfaceVisibility(const QString& backfaceVisibility) {}
 
-QVariantMap ImagePropertyHandler::source() const
-{
-  return QVariantMap{};
-}
+QVariantMap ImagePropertyHandler::source() const { return QVariantMap{}; }
 
 void ImagePropertyHandler::setSource(const QVariantMap& imageSource)
 {
-  QUrl source = imageSource["uri"].toUrl();
-  if (source.isRelative())
-    source = QUrl::fromLocalFile(QFileInfo(imageSource["uri"].toString()).absoluteFilePath());
-
-  m_bridge->imageLoader()->loadImage(source, [=](ReactImageLoader::Event event, const QVariantMap& data) {
-      if (event == ReactImageLoader::Event_LoadSuccess) {
-        m_object->setProperty("source", m_bridge->imageLoader()->provideUriFromSourceUrl(source));
-      }
-      if (m_onLoadEvents[event]) {
-        int reactTag = ReactAttachedProperties::get(qobject_cast<QQuickItem*>(m_object))->tag();
-        m_bridge->enqueueJSCall("RCTEventEmitter", "receiveEvent",
-                                QVariantList{reactTag, normalizeInputEventName(eventNames[event]), data});
-      }
+    QUrl source = imageSource["uri"].toUrl();
+    if (source.isRelative())
+        source = QUrl::fromLocalFile(QFileInfo(imageSource["uri"].toString()).absoluteFilePath());
+
+    m_bridge->imageLoader()->loadImage(source, [=](ReactImageLoader::Event event, const QVariantMap& data) {
+        if (event == ReactImageLoader::Event_LoadSuccess)
+        {
```

Comment:
@vkjr @narek-h I'm sorry for another one late feedback on formatting,
But how would you prefer if parentheses for `if conditions/loops/switches and so on` will start on the same line of code?
So it will look like:
if (event == ReactImageLoader::Event_LoadSuccess) {
    // Some code here
} else {
   // Some code here
}

Main argument is to have more valuable code in less screen space and it's the standard which is suggested , for example, by Google. Also, for example, Python language doesn't need parentheses at all (indents are used solely)

Also, can we have mandatory SPACE symbol between operators and expressions, for example:
`if`SPACE`(some_expression)`SPACE`{` ...

And also comments format, can we have it like this (SPACE symbol after // and starting with capital letter):

// Some comment here
// continue of comment if too long

Thanks

Reply:
@MaxRis, I'm ok with open brace on the same line.
Spaces in control expressions controlled by `SpaceBeforeParens: ControlStatements`, we already have it set.
Didn't find settings for spaces in comments.